src/components/Common/ResourceCategoryPicker.tsx

```
@@ -190,9 +190,14 @@ export function ResourceCategoryPicker({
   const getDisplayValue = () => {
     if (isLoadingSelected) {
       return (
-        <div className="flex items-center gap-2">
+        <div
+          className="flex items-center gap-2"
+          role="status"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<output>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)